packages/headless/src/modules/global-search/services/elastic.service.ts

```diff
@@ -44,12 +44,21 @@ async deleteSingleByElasticId(id: string): Promise<number> {
   /**
    * global product search - public api 
    */
-  async search(keyword: string, pageNumber=1, limit=10): Promise<IServiceResponse<ISearchProductResponse>>{
+  async search(keyword: string, pageNumber=1, limit=20): Promise<IServiceResponse<ISearchProductResponse>>{
     const result = await this.esHelperService.getProductSearchData(keyword, pageNumber, limit )
     if(!result){
       return errorResponse("Error while searching", null, HttpStatus.CONFLICT)
     }
     return successResponse(null, result, HttpStatus.OK)
   }
   
+  
+  async searchSuggestion(keyword: string): Promise<IServiceResponse<ISuggestedProductResponse>>{
+    const result = await this.esHelperService.getProductSearchSuggestion(keyword)
+    if(!result){
+      return errorResponse("Error while searching", null, HttpStatus.CONFLICT)
+    }
+    return successResponse(null, result, HttpStatus.OK)
```

Comment:
successResponse function takes two arguments only.

Reply:
First argument was required if we want to validate response. At this moment this can not be change because this pattern is using in other modules.